### DISCUSSION TITLE
trying to join the SF Bay server.....not getting confirmation emails

### DISCUSSION THREAD

**Discussion by neutrino78x** [mastodon/mastodon#20266] [General]
So I'm trying to start a new account. I created one on the SF Bay Area server but it says it sent a confirmation email and I never got it. I have tried to have it resend several times.

Are all the servers experiencing this or would trying again on another server help?

**Comment by MrGeneration** [comment]
This doesn't have to happen to all servers.
It depends on the load and configuration of the affected server.

If you can't find the mail in your spam folder, there may be issues during delivery to you (of whatever sort).

If you don't want to be patient and wait, you could look for another -to you- fitting instance or contact the instance admin (you'll find contact info on the `/about` page).
